backend/typescript/rest/camperRoutes.ts

```diff
@@ -105,11 +106,24 @@ camperRouter.post(
   },
 );
 
+camperRouter.put("/waitlist/:waitListCamperId", async (req, res) => {
+  try {
+    const updatedWaitlistedCamper = await camperService.updateWaitlistedCamperStatus(
+      req.params.waitListCamperId,
+    );
+    res.status(201).json(updatedWaitlistedCamper);
```

Comment:
should be status 200 for success, 201 is for successful resource creation? according to google 🤷‍♀️ 
```suggestion
    res.status(200).json(updatedWaitlistedCamper);
```